File: backends/php-laravel/app/Domain/Client/Condition/JwtHistoryCondition.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\Domain\Client\Condition;

use App\Support\Repositories\Conditions\AbstractCondition;

/**
 * JWT履歴Conditionクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Client\Condition
 */
class JwtHistoryCondition extends AbstractCondition
{
    public ?int $clientId = null;
}

File: backends/java-spring/src/main/java/com/authorization/domain/client/condition/JwtHistoryCondition.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.client.condition;

import com.authorization.support.repositories.conditions.AbstractCondition;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * JWT履歴Conditionクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class JwtHistoryCondition extends AbstractCondition {

    private Long clientId;
}
